docs/theia-cli.md

```diff
@@ -36,14 +36,20 @@ theia help
 
 ## Usage
 
-To see the list of available commands and options, run `theia help`. Currently,
-we have 5 commands for the NetworkPolicy Recommendation feature:
+To see the list of available commands and options, run `theia help`.
+We currently have below commands for NetworkPolicy Recommendation and ClickHouse:
+
+NetworkPolicy Recommendation:
 
 - `theia policy-recommendation run`
 - `theia policy-recommendation status`
 - `theia policy-recommendation retrieve`
 - `theia policy-recommendation list`
 - `theia policy-recommendation delete`
 
+ClickHouse:
+
+- `theia clickhouse status`
+
 For details, please refer to [NetworkPolicy recommendation doc](
 networkpolicy-recommendation.md)
```

Comment:
Maybe move this paragraph above the ClickHouse section.

Reply:
Updated. Thanks.